tutorials.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <meta name="description" content="">
+    <meta name="author" content="">
+
+    <title>SciSpark Website</title>
+
+        <!-- Google Fonts -->
+
+        <!-- Bootstrap core CSS -->
+        <link href="bower_components/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
+        <link href="styles.css" rel="stylesheet">
+
+        <!-- Favicon -->
+
+        <!-- Bootstrap core JavaScript
+        ================================================== -->
+        <!-- Placed at the end of the document so the pages load faster -->
+        <script src="bower_components/jquery/dist/jquery.min.js"></script>
+        <script src="bower_components/bootstrap/dist/js/bootstrap.min.js"></script>
+        <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
+        <!-- <script src="../../assets/js/ie10-viewport-bug-workaround.js"></script>-->
+
+        <script>
+        $(function(){
+          $("#header").load("header.html");
+          $("#footer").load("footer.html");
+        });
+        </script>
+
+  </head>
+
+  <body>
+
+
+    <a class="accessible" href="#maincontent">[Skip to Content]</a>
+
+    <!-- Note: header content is in header.html and pulled in by jquery script in the head -->
+    <div id="header"></div>
+
+
+      <div class="container">
+        <a name="maincontent"></a>
+
+        <h1> Tutorials </h1>
+
+        <p>We initially developed these tutorials for a full day of training at <a href="http://esipfed.org/">ESIP</a> Summer 2016. We have improved on them since then, and welcome any additional suggestions you may have.
+        </p>
+        <p>The training materials include PDFs of our presentations, a docker file so you can install and work on SciSpark, and a collection of Notebooks for SciSpark with training examples you can follow and work through.
+        </p>
+
+        <h2> Presentations (PDFs) </h2>
```

Comment:
Ack'ed Wayne. 
Maybe something like 
"The training materials include our presentations, a docker file so you can install and work on SciSpark, and a collection of Notebooks for SciSpark with training examples you can follow and work through.

PRESENTATIONS 
.."
There is rubric to indicate that Adobe reader is needed. So you could refrain from the using the acronym.